hycom/MSCPROGS/src/Nersclib/mod_hycomfile_io.F90

```diff
@@ -1244,10 +1244,10 @@ subroutine netcdfInfo(vnamein,gridrotate,stdname,longName,units,vname,cellmethod
          stdname='sea_ice_area_fraction' ; units='1' ; vname='siconc'
          limits=(/0,1/)
       case ('fy_age','iage_d')
-          stdname='age_of_sea_ice' ; units='year' ; vname='siage'
-          limits=(/0,365/)
+          stdname='age_of_sea_ice' ; units='day' ; vname='siage'
+          limits=(/0,36500/)
        case ('fy_frac','FYarea_d')
-          stdname='sea_ice_fraction_of_first_year' ; units='1' ;  
+          stdname='sea_ice_area_fraction_of_first_year_ice' ; units='1' ;  
```

Comment:
It seems that this topic was left hanging. 
I found this discussion and code from Knut Arild (2016) about sea ice classes,
can you implement it? 
(note that Elodie is now your colleague at MET Norway... )

On 20 Jul 2016, at 11:34, Elodie Fernandez <elodie.fernandez@mercator-ocean.fr> wrote:

Dear Knut and Laurent,

I'm getting back to you about the sea-ice table. In my previous email (01/07), I said that only one variable from this table didn't have a standard_name, the "fraction_of_first_year_ice". Somehow, I completely missed the "age_of_first_year_ice" which is not standard either. The MFC Arctic is the only producer of these variables, so I think it's easier to discuss that together directly.

I looked into your suggestion Knut and also looked back at the CF list archives and found the old discussion about these names (back in 2011!). I think the structure you suggested is right. Do you also agree that the same should be done for "age_of_first_year_ice": use the existing standard_name "age_of_sea_ice" and add the cell_method "area: mean where siclass"? The suggested unit from CF is "years" but we can keep "days" as it's what you already use. 

To be consistent with the other names, I suggest the following names:
- sea_ice_area_fraction "where first year ice" -> siconc_fy
- sea_ice_age "where first year ice" -> siage_fy

What do you think?

Regards,
Elodie

On 06/07/2016 08:34, "Knut A. Lisæter" wrote:
Dear Elodie,

I’m sure you have put some effort into this,  so thank you for your work. 

I have a comment/suggestion about the sea-ice reference table: specifically the variable sifyfrac, or fraction_of_first_year_ice

I believe this variable can also be described by using the auxiliary table with standard name “sea_ice_classification”.  The structure would be something like the 

dimensions:
  lat=73;
  lon=96;
  maxlen=20;
  ls=1;
variables:
  float sifyfrac(ls,lat,lon);
    sifyfrac:coordinates="siclass";
    sifyfrac:cell_methods="area: mean where siclass";
    sifyfrac:standard_name="sea_ice_area_fraction";
  char siclass(ls,maxlen);
    siclass:standard_name="sea_ice_classification";
data:
  siclass="first_year_ice";


The above method is described in some detail in the cf metadata document (section 7.3.3: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#statistics-applying-portions ), with area_type replaced by sea_ice_classification.

This would make the dataset somewhat harder to use (there is an extra dimension in the “sifyfrac” variable), but it should be as close to being CF-compliant as we can get. On the other hand, if you try to approach the CF group to suggest a new standard name fraction_of_first_year_ice”, I believe they will point you to the solution above (there has been some discussion on this topic already on the CF mailing list)...